release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,11 +6,24 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 4 December 2023 - New Feature - DataMiner as a Service [ID_XXXXXXXXXXXXXXXXXXX]
+
+From now onwards it is possible to create a DataMiner System as a Service on dataminer.services, in short called DaaS. 
+Deploying such a DaaS will deduct 3 DataMiner credits from the organization. Every week onwards another 3 DataMiner Credits will be deducted unless the organization runs out of DataMiner Credits. In that case the particulaar DaaS instance will be deleted.
+When a DaaS is created, the dataminer.services account and the DataMiner account will automatically be linked to each each other, so the web apps like Monitoring are easily available.
```

Comment:
Not sure if it's worth mentioning here, it'd be if this previously wasn't the case. But this is the for change log for DaaS